pkg/clusters/addons/kong/vars.go

```diff
@@ -42,4 +42,7 @@ const (
 
 	// DefaultUDPServicePort indicates the default open port to be found on the Kong proxy's UDP service.
 	DefaultUDPServicePort = 9999
+
+	// DefaultUDPServicePort indicates the default open port that will be used for TCP traffic.
+	DefaultTCPServicePort = 8888
```

Comment:
While we're here, can we add DefaultTLSServicePort 8899 in `kong/addon.go` also, assuming we'll need it imminently for TLSRoute?

Once again, I am mad at GH for not allowing suggestions to extend onward into code outside the diff region.

```suggestion
	// DefaultTCPServicePort indicates the default open port that will be used for TCP traffic.
	DefaultTCPServicePort = 8888
	
	// DefaultTLSServicePort indicates the default open port that will be used for TLS traffic.
	DefaultTLSServicePort = 8899
```